src/Date/DateFormat.php

```diff
@@ -37,4 +37,5 @@ class DateFormat
     const FORMAT_MYSQL_STYLE = 'Y-m-d H:i:s';
     const FORMAT_DAY = 'jS';
     const FORMAT_LONG = 'jS F Y';
+    const FORMAT_DATE_TIME_LONG = 'jS F Y \a\t h:iA';
```

Comment:
This constant's name should be something like `FORMAT_LONG_DATE_AT_TIME` so we know what the format looks like at a glance